aim/web/ui/src/pages/Metrics/MetricsContainer.tsx

```diff
@@ -75,6 +77,7 @@ function MetricsContainer(): React.FunctionComponentElement<React.ReactNode> {
     if (route.params.appId) {
       appRequestRef = metricAppModel.getAppConfigData(route.params.appId);
       appRequestRef.call().then(() => metricAppModel.getMetricsData().call());
+      metricAppModel.setDefaultAppConfigData();
```

Comment:
if we use this method for if/else cases why we need to write it in if/else block, we can take out from this block if i'm not mistaken

Reply:
yeah, you're right, I actually should have put it right after `metricAppModel.getMetricsData().call()` inside promise then